src/types.jl

```diff
@@ -135,6 +135,11 @@ blocklen(::Type{SHA3_256_CTX}) = UInt64(25*8 - 2*digestlen(SHA3_256_CTX))
 blocklen(::Type{SHA3_384_CTX}) = UInt64(25*8 - 2*digestlen(SHA3_384_CTX))
 blocklen(::Type{SHA3_512_CTX}) = UInt64(25*8 - 2*digestlen(SHA3_512_CTX))
 
+rate_in_bytes(::Type{SHA3_224_CTX}) = 144
+rate_in_bytes(::Type{SHA3_256_CTX}) = 136
+rate_in_bytes(::Type{SHA3_384_CTX}) = 104
+rate_in_bytes(::Type{SHA3_512_CTX}) = 72
+
```

Comment:
```suggestion
```

It seems that these functions are not used.

Reply:
Ah, Yes, I was able to simplify. Thank you!